gmsh_utils/gmsh_IO.py

```diff
@@ -350,8 +350,9 @@ def generate_geometry(self, point_coordinates: Union[List[List[List[float]]], np
             None
         """
 
-        gmsh.initialize()
-        gmsh.model.add(mesh_name)
+        if not gmsh.isInitialized():
```

Comment:
why is it necessary to check if gmsh is initialized or not?

Reply:
Because I want to call this function multiple times, without reseting the gmsh data, for reference see this (draft) Pr:

https://github.com/StemVibrations/STEM/pull/56